app/templates/auth/force_change_password.html

```
@@ -0,0 +1,40 @@
+{% extends "base.html" %}
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)